src/google/protobuf/util/json_util_test.cc

```diff
@@ -457,6 +458,20 @@ TEST(ZeroCopyStreamByteSinkTest, TestAllInputOutputPatterns) {
   }
 }
 
+TEST_F(JsonUtilTest, TestWronJsonInput) {
+  const char json[] = "{\"unknown_field\":\"some_value\"}";
```

Comment:
Couple small things:
- Add line breaks to keep the lines under 80 characters long
- Our convention is to use `snake_case` for variables in C++ so `protoBuffer` should be `proto_buffer` and likewise for `messageType`, and `resultStatus`.